src/codegen/utilities/interpreter_helpers.py

```diff
@@ -34,6 +34,10 @@
     "SetFirstSampClkWhen",
     "SetStartTrigTrigWhen",
     "SetSyncPulseTimeWhen",
+    "ReadPowerBinaryI16",
+    "ReadPowerF64",
+    "ReadRaw",
+    "WriteRaw",
```

Comment:
This breaks GrpcStubInterpreter. If the method is defined in BaseInterpreter but not GrpcStubInterpreter, then you can't instantiate GrpcStubInterpreter.

Here's how to fix it:
- Move these to LIBRARY_INTERPRETER_IGNORED_FUNCTIONS
- Remove the hardcoded versions from _base_interpreter.py.mako

Reply:
Sure Brad. 

- I have added those functions in `LIBRARY_INTERPRETER_IGNORED_FUNCTIONS` and skip their generation in `_library_interpreter.py.mako`. 
- Also removed the hardcoded versions from `_base_interpreter.py.mako` .
- Changed `python_codegen_method` of those functions from '`no`' to '`Custom_Code_Read_Write`.
please say if this is fine or not?

Reply:
Yes, Completely agree on your thoughts. All these parameters changes will be handled in the upcoming PRs. Thanks for pointing this out.

Reply:
Looks good